pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentUploadDownloadRestletResource.java

```diff
@@ -555,6 +750,67 @@ public void uploadSegmentAsMultiPart(FormDataMultiPart multiPart,
     }
   }
 
+  @POST
+  @ManagedAsync
+  @Produces(MediaType.APPLICATION_JSON)
+  @Consumes(MediaType.MULTIPART_FORM_DATA)
+  @Path("/segmentList")
```

Comment:
Suggest making the API path `segments/{tableName}/batchUpload`. `POST /segmentList` is quite misleading to me

Reply:
maybe consider `v3/segments` as there is `v2/segments`

Reply:
I'm not a big fan of `v2`, `v3` etc. I prefer having a separate API just for batch upload

Reply:
we should document the convention for naming api's. +1 on /segments. upload is a verb and it should not be part of the URI.

https://medium.com/@nadinCodeHat/rest-api-naming-conventions-and-best-practices-1c4e781eb6a5